chain-connect/src/GalachainConnectClient.ts

```diff
@@ -22,45 +22,45 @@ declare global {
 }
 
 export class GalachainConnectClient {
-  private address_: string;
-  private provider_: BrowserProvider | undefined;
-  private chainCodeUrl: string;
+  #address: string;
+  #provider: BrowserProvider | undefined;
+  #chainCodeUrl: string;
 
   get address() {
-    return `eth|${this.address_}`;
+    return `eth|${this.#address}`;
   }
 
   set address(val: string) {
-    this.address_ = val.replace(/0x|eth\|/, "");
+    this.#address = val.replace(/0x|eth\|/, "");
   }
 
   get provider() {
-    return this.provider_;
+    return this.#provider;
   }
 
   constructor(chainCodeUrl: string) {
-    this.chainCodeUrl = chainCodeUrl;
+    this.#chainCodeUrl = chainCodeUrl;
   }
 
   public async connectToMetaMask() {
     if (!window.ethereum) {
       throw new Error("Ethereum provider not found");
     }
 
-    this.provider_ = new BrowserProvider(window.ethereum);
+    this.#provider = new BrowserProvider(window.ethereum);
 
     try {
-      const accounts = (await this.provider_.send("eth_requestAccounts", [])) as string[];
-      this.address_ = accounts[0];
+      const accounts = (await this.#provider.send("eth_requestAccounts", [])) as string[];
+      this.address = accounts[0];
 
-      return this.address;
+      return this.#address;
```

Comment:
This should still just return this.address I think so the user is returned the valid GalaChain address